File: outbound-call.py
import os
from twilio.rest import Client
account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']
client = Client(account_sid, auth_token)

call = client.calls.create(
    twiml='<Response><Say>AUTO WARRANTY INSURANCE</Say></Response>',
    to=os.environ['MY_PHONE_NUMBER'], 
    from_='+12672042695'
    )

print(call.sid)

File: outbound-call.js
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const client = require('twilio')(accountSid, authToken);
client.calls.create({
  twiml: '<Response><Say>Hello thank you for picking up</Say></Response>',
  to: process.env.MY_PHONE_NUMBER,
  from: 'TWILIO-PHONE-NUMBER'
}).then(call => console.log(call.sid))